app/src/pages/Player/components/AchievementGroup/AchievementGroup.jsx

```diff
@@ -89,8 +89,9 @@ function formatThreshold(threshold) {
     return threshold;
   }
 
-  if ([273742, 737627, 1986068, 5346332, 13034431].includes(threshold)) {
-    return getLevel(threshold + 100).toString();
+  const totalSkills = SKILLS.filter(s => s !== 'overall').length;
```

Comment:
you can import `REAL_SKILLS` from `@wise-old-man/utils` and use that instead of filtering for != overall